tools/launcher/common/eagle3/dump_offline_data_hf.sh

```
@@ -0,0 +1,55 @@
+# Suitable for: VLMs, models with custom code, architectures not yet in TRT-LLM.
+#
+# Required environment:
+#   HF_MODEL_CKPT   Path to the HF model checkpoint
+#
+# Args passed through to compute_hidden_states_hf.py:
+#   --input-data, --output-dir, --max-seq-len, etc.
+###################################################################################################
+
+pip install datasets 2>/dev/null || true
+
+if [ -z ${SLURM_ARRAY_TASK_ID} ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)